src/library_gl.js

```diff
@@ -740,11 +740,12 @@ var LibraryGL = {
         break;
       case 0x8B8C /* GL_SHADING_LANGUAGE_VERSION */:
         var glslVersion = GLctx.getParameter(GLctx.SHADING_LANGUAGE_VERSION);
-        // Map WebGL GL_SHADING_LANGUAGE_VERSION string format to GLES format.
-        if (glslVersion.indexOf('WebGL GLSL ES 1.0') != -1) glslVersion = 'OpenGL ES GLSL ES 1.00 (WebGL)';
-#if USE_WEBGL2
-        else if (glslVersion.indexOf('WebGL GLSL ES 3.00') != -1) glslVersion = 'OpenGL ES GLSL ES 3.00 (WebGL 2)';
-#endif
+        // extract the version number 'N.M' from the string 'WebGL GLSL ES N.M ...'
+        var ver_num = glslVersion.substr(14);
```

Comment:
Let's instead use a regex to match to this expected form "WebGL GLSL ES N.M" -> "OpenGL ES GLSL ES N.M (WebGL [2])", and not use a hardcoded 14 characters length here and indexOf/substr string manipulation, since that feels brittle. If the regex fails (browser reports something out of line), then report the original glslVersion string?